gvm/protocols/gmpv208/gmpv208.py

```diff
@@ -177,56 +177,6 @@ def get_info(self, info_id: str, info_type: InfoType) -> Any:
         cmd.set_attribute("details", "1")
         return self._send_xml_command(cmd)
 
-    def modify_schedule(
```

Comment:
`modify_schedule` got removed. was this by intention?

Reply:
Yes. I have copied it earlier into that file. But it was not different to the earlier version in gmpv9. So I removed it again ... I told you earlier ...

Reply:
Ah thanks for the info. Would be best to split such thing at least into another commit and at best into anther pr